mcs/class/System/System.Security.Cryptography.X509Certificates/X509Certificate2.cs

```diff
@@ -216,46 +218,32 @@ public string GetNameInfo (X509NameType nameType, bool forIssuer)
 
 		public override void Import (byte[] rawData) 
 		{
-			Import (rawData, (string)null, X509KeyStorageFlags.DefaultKeySet);
+			base.Import (rawData);
 		}
```

Comment:
No need to override at all, same with the methods below.

Reply:
I verbosely copied these from CoreFX's version.